mssqlcli/mssqlclioptionsparser.py

```diff
@@ -187,3 +191,23 @@ def create_parser():
     )
 
     return args_parser
+
+def check_positive_int(row_limit):
```

Comment:
Can you add 'row_limit' to the method name? This way, it stands out from being a generic 'check_positive_int' method. Ideally, we have a generic method where check_row_limit calls check_positive_int('row_limit' value), and check positive can be resused.